views/index.jade

```diff
@@ -14,12 +14,12 @@ html
       
       #question-div
         
-        h2#question
+        h2#question Loading...
         #timer
         
       #input-container
         .input-div
-          input#answer-input(type='text', placeholder='post an answer')
+          input#answer-input(type='text', disabled, placeholder='no question to answer')
```

Comment:
Again, this is the new default, but it'll quickly get set to `post an answer` if there's a question.